game/templates/main-menu.html

```diff
@@ -1,28 +1,90 @@
+<html>
 {% load socialaccount %}
 {% providers_media_js %}
 {% load static %}
-
-<link rel="stylesheet" type="text/css" href='{% static "/css/style.css" %}'>
 <head>
-    <title>Login Page</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<title>Main Menu</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="stylesheet" type="text/css" href='{% static "/css/menu-style.css" %}'>
 </head>
-
-<body>
-    <p>User: {{user}}</p>
-    <p>{{ question.answer }}</p>
+<body class="header_area">
+    <div class="main_title">
+        <h2>Riddle Me This</h2>
+        <h1>Riddle Me This</h1>
+        <a class="video-play-button">
+            <span>?</span>
+        </a>
+    </div>
+    <div class="navbar">
+        <ul class="nav navbar-nav navbar-right">
+            <div style="text-align: center;">
+                <li class="nav-item submenu">
+                    <a class="primary_btn">Play</a>
+                        <ul class="dropdown-menu">
+                            <li class="nav-item"><a href="{% url 'game:easy' %}" class="nav-link">Easy</a>
+                            <li class="nav-item"><a href="{% url 'game:medium' %}" class="nav-link">Medium</a>
+                            <li class="nav-item"><a href="{% url 'game:hard' %}" class="nav-link">Hard</a>
+                        </ul>
+                </li>
+            </div>
+            <br></br>
+            <li class="nav-item"><a href="{% url 'game:htp' %}" class="primary_btn">How to Play</a></li>
+            <br></br>
+            <li class="nav-item"><a href="#" class="primary_btn">Scoreboard</a></li>
+        </ul>
+    </div>
     {% if user.is_authenticated %}
-    <h1 style="text-align: center;">ON GAME PAGE</h1>
-    <a href="{% url 'game:hard' %}">to stage</a>
-    <button id='logout-bth'><a href="{% url 'logout' %}">logout</a></button>
+    <div id="mySidenav" class="sidenav">
+        <a href="javascript:void(0)" class="closebtn" onclick="closeNav()"><i class="fa fa-window-close"></i></a>
+        <a class="sidetext" style="color: #effeff;"> Welcome, {{user}}</a>
+        <div class="log">
+            <div class="l_btn">
+                <a id='logout-bth' class="log_btn" href="{% url 'logout' %}"><span>Logout</span></a>
+            </div>
+        </div>
+    </div>
+    <span id="opennav" class="side_bar" onclick="openNav(), style.display = 'none'"><i class="fa fa-chevron-circle-left"></i></span>
+    <div class="footer">
+        <h3>You are now logged in as {{user}}. Enjoy the game.  <i class="fa fa-hand-peace-o"></i></h3>
```

Comment:
it would be nice if we can change user to a Oauth provider.

Reply:
Thank you for suggesting. 🙇‍♀ 
I think we can import it from the user-forms branch to display more of user profile.

Reply:
Sounds good, I will try to update profile. thanks for sharing new ideas :D